test/verify/check-networking

```diff
@@ -419,7 +419,7 @@ class TestNetworking(MachineCase):
 
         # Create a tun device and let NetworkManager manage it
         m.execute("ip tuntap add mode tun dev %s" % iface)
-        m.execute("nmcli --wait 0 dev con %s" % iface)
+        wait(lambda: m.execute("nmcli --wait 0 dev con %s" % iface) == "")
```

Comment:
The activation will not succeed fully.  The interface will not receive a IP address, for example.  We just want NM to recognize this interface and mark it as "managed".